### PR TITLE
Added multiple device support under features

### PR DESCRIPTION
**Pull request by smash-cracker**
Added multiple device support under features in readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,6 +13,8 @@ A WhatsApp Desktop client for linux systems. This is an unofficial client. Use h
 <hr/>
 
 ## Features
+- **WhatsApp Multi Device** 📱🖥️  
+Use whatsApp in multiple device
```
Comment (YoungFellow-le) [inline README.md:17]: I recommend you reword this to:
```
Full support for the new WhatsApp Multi Device feature!
```

Comment (smash-cracker) [reply README.md:17]: okay. thanks for suggestion

Comment (YoungFellow-le) [reply README.md]: You're welcome :)
